virtool/downloads/api.py

```diff
@@ -58,6 +58,28 @@ async def download_analysis(req):
     return web.Response(text=formatted, headers=headers)
 
 
+@routes.get("/download/hmms/profiles.hmm")
+async def download_hmm_profiles(req):
+    """
+    Download the HMM profiles file if HMM data is available.
+
+    """
+    file_path = Path(req.app["settings"]["data_path"]) / "hmm" / "profiles.hmm"
+
+    if not file_path.parent.is_dir():
+        return virtool.api.response.not_found("HMM data could not be found")
+
+    if not file_path.is_file():
+        return virtool.api.response.not_found("Profiles file could not be found")
```

Comment:
Not necessary to return different nessages. `HMM data could not be found` is not very informative. Use `Profiles file could not be found` in both cases.

Combine these checks and extract into function in `hmm.utils`. Then, run in thread to avoid blocking. Please double-check that a function like this doesn't already exist.